app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -137,14 +134,37 @@ public void saveViewState(Bundle outState) {
     }
 
     @Override
-    public void setBlockingEnabled(boolean enabled) {
+    public void setContentBlockingEnabled(boolean enabled) {
+        if (client.isBlockingEnabled() == enabled) {
+            return;
+        }
+
         client.setBlockingEnabled(enabled);
+
+        if (!enabled) {
+            reloadOnAttached();
+        }
     }
 
     public boolean isBlockingEnabled() {
         return client.isBlockingEnabled();
     }
 
+    @Override
+    public void setImageBlockingEnabled(boolean enabled) {
+        WebSettings settings = getSettings();
+        if (enabled == settings.getBlockNetworkImage() &&
+                enabled == !settings.getLoadsImagesAutomatically()) {
```

Comment:
please move `&&` to the start of next line. Maybe to use `flag` is better than `enabled`? (since enabled indicate a happened-state of WebSettings)

Reply:
already moved the operator to the head of the second line.